ui/ui-components/pages/object/[id].tsx

```diff
@@ -123,21 +121,7 @@ export default function FindObject(props) {
     };
   }
 
-  function getListPage(tableName: string) {
-    const byModel = [
-      "records",
-      "groups",
-      "destinations",
-      "sources",
-      "properties",
-      "schedules",
-    ];
-    if (byModel.includes(tableName) && props.navigationModel.value) {
-      return `/model/${navigationModel.value}/${tableName}`;
-    }
-
-    return `/${tableName}`;
-  }
+  const getListPage = (tableName: string) => `/${tableName}`;
```

Comment:
I believe this means that in community edition, where we show list pages instead of edit pages for most "tables", it would link to something like `/groups` or `/properties`, which would 404. Let's check what the behavior is for going to `/object/some_group_id` in community edition after this change and ensure it still works. I think some additional change would need to be made to instead take them to the model page or something like that.

Reply:
Ok. I need to rethink how this is done right now because I will need to resolve the model id some other way.